bundles/binding/org.openhab.binding.comfoair/src/main/java/org/openhab/binding/comfoair/handling/ComfoAirConnector.java

```diff
@@ -423,4 +437,15 @@ private String dumpData(byte[] data) {
         }
         return sb.toString();
     }
+
+    private class CPUWorkaroundThread implements SerialPortEventListener {
+        @Override
+        public void serialEvent(SerialPortEvent event) {
+            try {
+                logger.trace("RXTX library CPU load workaround, sleep forever");
+                Thread.sleep(Long.MAX_VALUE);
+            } catch (InterruptedException e) {
+            }
+        }
+    }
```

Comment:
I'm concerned about this workaround, only because I'm unaware of other bindings that use the serial port having to implement a workaround of this sort.  Can you offer how this binding's use of the serial port is necessarily different from, say, how the Z-Wave binding uses it?